### ISSUE TITLE
Accessibility of submit button

### ISSUE THREAD

**Issue by eddogan** [wpninjas/ninja-forms#3659]
Submit button type should be set as "submit" not "button" according to this specification for A level compliance: 

https://www.w3.org/TR/WCAG20-TECHS/H32.html

**Comment by omikron** [comment]
This has always seemed strange to me that the submit is not type submit. Is there a reason or this?

**Comment by codemonkeynorth** [comment]
@eddogan @omikron can you confirm it's also not tabbable with a keyboard? on my form the tab focus skips over the button
